pmda/rmsf.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import MDAnalysis as mda
+import pmda
+import dask
+from dask.delayed import delayed
+import time
+from joblib import cpu_count
+from pmda.parallel import ParallelAnalysisBase
+from pmda.util import timeit
+
+class RMSF(ParallelAnalysisBase):
+    """Parallel RMSF analysis.
+
+    Attributes
+    ----------
+    Parameters
+    ----------
+
+    Note
+    ----
+    At the moment, this class doesn't do anything.
+    """
+
+    def __init__(self, atomgroup):
+        u = atomgroup.universe
+        super(RMSF, self).__init__(u, (atomgroup,))
+        self._atomgroup = atomgroup
+        self._top = u.filename
+        self._traj = u.trajectory.filename
+
+    def _prepare(self):
+        self.sumsquares = np.zeros((self._atomgroup.n_atoms, 3))
+        self.mean = np.zeros((self._atomgroup.n_atoms, 3))
+
+    def _single_frame(self, ts, agroups, frame_index, sumsquares, mean):
+        """
+        Called for every frame.
+
+        Parameters
+        ----------
+        ts : int
+            current time step
+        agroups :
+            tuple of atomgroups that are updated to the current frame
+
+        Returns result of RMSF function for a single frame.
+        """
+        k = frame_index
+        # self.sumsquares += (k / (k+1)) * (self.atomgroup.positions - self.mean) ** 2
+        sumsquares += (k / (k+1)) * (agroups.positions - mean) ** 2
+        mean = (k * mean + agroups.positions) / (k + 1)
```

Comment:
If `agroups` is a tuple of ATomGroups the `agroups.positions` will throw an attribute error because a `tuple` does not have `.positions`. 

See my comment on docs above...